### DISCUSSION TITLE
Active tab title style

### DISCUSSION THREAD

**Discussion by mapsgeek** [mantinedev/mantine#3841] [Q&A] [answered]
I need an example on how to give the current active tab a different style?

**Comment by Sajarin-M** [comment, marked as answer]
Check out this [example](https://ui.mantine.dev/category/headers#header-tabs-colored) from Mantine UI.
Here, the `data-active` attribute on the tab is used to style the active tab.
```js
tab: {
  '&[data-active]': {
     /* active tab styles */
  },
}
```
